scripts/setup-project.sh

```diff
@@ -126,6 +126,13 @@ create_threadx_projects() {
       project_ide_dir='mplab/'
       libs="same54_lib filex common_hardware_code "
       ;;
+    ck-rx65n)
+      echo Downloading Azure_RTOS_6...
+      wget -q -O azrtos.zip https://saleshosted.z13.web.core.windows.net/sdk/AzureRTOS/Azure_RTOS_6.1_RX65N_Cloud_Kit_E2Studio_GNURX_Samples_2022_05_25.zip
+      project_platform_dir=''
+      project_ide_dir='e2studio_gnurx/'
+      libs="filex netxduo_addons "
+      ;;
     rx65ncloudkit)
       echo Downloading Azure_RTOS_6...
       wget -q -O azrtos.zip https://saleshosted.z13.web.core.windows.net/sdk/AzureRTOS/Azure_RTOS_6.1_RX65N_Cloud_Kit_E2Studio_GNURX_Samples_2022_05_25.zip
```

Comment:
Crossing patches -- need to use ${FETCH} instead of "wget -q -O" when rebase / merge back to main.
See recent changes to serup-project.sh